README.md

```diff
@@ -74,9 +74,10 @@ To join in:
 2. Apply to join our [mailing list](https://groups.google.com/a/chromium.org/forum/#!forum/message-format-wg)
 3. Watch this repository (use the "Watch" button in the upper right corner)
 
-## Copyright & Licenses
+### Copyright & Licenses
 
-Copyright © 1991-2023 Unicode, Inc. Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the United States and other countries.
+Copyright © 2019-2023 Unicode, Inc. Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the United States and other countries.
 
-The project is released under [LICENSE](./LICENSE). A CLA is required to contribute to this project - please refer to the CONTRIBUTING.md file (or start a Pull Request) for more information.
+The project is released under [LICENSE](./LICENSE).
 
+A CLA is required to contribute to this project - please refer to the [CONTRIBUTING.md](https://github.com/unicode-org/.github/blob/main/.github/CONTRIBUTING.md) file (or start a Pull Request) for more information.
```

Comment:
no CONTRIBUTING.md in this repo?

Reply:
We do have one, and it includes the CLA instructions.

```suggestion
A CLA is required to contribute to this project - please refer to the [CONTRIBUTING.md](./CONTRIBUTING.md) file (or start a Pull Request) for more information.
```

Reply:
@eemeli my mistake, will fix

Reply:
PTAL